tests/macros/m18235.nim

```diff
@@ -17,4 +17,26 @@ macro unexport(n: typed): untyped =
   result[0] = n.name.strVal.ident
 
 proc foo*() {.unexport.} = discard
-proc bar() {.rename.} = discard
\ No newline at end of file
+proc bar() {.eexport.} = discard
+
+proc foooof*() {.unexport, eexport, unexport.} = discard
+proc barrab() {.eexport, unexport, eexport.} = discard
+
+macro eexportMulti(n: typed): untyped =
+  # use the call version of `eexport` macro for one or more decls
+  result = copyNimTree(n)
+  for i in 0..<result.len:
+    result[i] = newCall(ident"eexport", result[i])
+
+macro unexportMulti(n: typed): untyped =
+  # use the call version of `unexport` macro for one or more decls
+  result = copyNimTree(n)
+  for i in 0..<result.len:
+    result[i] = newCall(ident"unexport", result[i])
+
+unexportMulti:
+  proc oof*() = discard
```

Comment:
```suggestion
  proc oof*() = discard
  proc oof2() = discard
```
(optional; it case it also handles this easily)

Reply:
unexport doesnt handle this